src/storage/txn/commands/mod.rs

```diff
@@ -925,11 +931,20 @@ pub mod test_util {
             extra_op: ExtraOp::Noop,
             statistics,
             async_apply_prewrite: false,
+            apiv2_ctx: None,
         };
 
         let ret = cmd.cmd.process_write(snap, context)?;
         let ctx = Context::default();
         engine.write(&ctx, ret.to_be_write).unwrap();
         Ok(())
     }
+
+    pub fn get_apiv2_ctx(api_version: ApiVersion) -> Option<ApiV2Context> {
```

Comment:
Suggest to name as `mock_raw_ext`, to make it explicitly different with the real one.